assets/js/popup.js

```diff
@@ -0,0 +1,136 @@
+  {
+    title: 'Tonic',
+    img: 'assets/images/desktop-images/desktop-work1.png',
+    description: 'A daily selection of privately perosonalized reads; no accounts or sign-ups required.',
+    pageLink: 'https://github.com/Bria222/Bria222.github.io',
+    sourcesLink: 'https://github.com/Bria222/Bria222.github.io',
+  },
+  {
+    title: 'Multi-Post Stories',
+    img: 'assets/images/desktop-images/desktop-work2.png',
+    description: 'Experimental content creation feature that allows users toadd to an existing story over the course of a day without spamming their friends.',
+    pageLink: 'https://github.com/Bria222/Bria222.github.io',
+    sourcesLink: 'https://github.com/Bria222/Bria222.github.io',
+  },
+  {
+    title: 'Facebook 360',
+    img: 'assets/images/desktop-images/desktop-portfolio-image1.png',
+    description: 'Exploring the future of media in Facebooks first VirtualReality app; a place to discover and enjoy 360 photos andvideos on Gear VR.',
+    pageLink: 'https://github.com/Bria222/Bria222.github.io',
+    sourcesLink: 'https://github.com/Bria222/Bria222.github.io',
+  },
+  {
+    title: 'Uber Navigation',
+    img: 'assets/images/desktop-images/desktop-portfolio-image2.png',
+    description: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    pageLink: 'https://github.com/Bria222/Bria222.github.io',
+    sourcesLink: 'https://github.com/Bria222/Bria222.github.io',
+  }];
+// Modal projects event
+// First Modal
+projectBtn.forEach((btn, index) => {
+  btn.addEventListener('click', () => {
+    printModal(`<div class="modal-header-container">
+    <div class="modal-title-container">
+      <h2 class="modal-title">${modalProject[index].title}</h2></div>
+      <div class="modal-header-icon">
+        <i id ="close-menu" class="fas fa-times"></i>
+      </div>
+    </div>
+    <div class="modal-subtitle-container">
+      <ul class="modal-mark">
+        <li>
+          <p>CANOPY</p>
+        </li>
+        <li>
+          <p>Back End Dev</p>
+        </li>
+        <li>
+          <p>2015</p>
+        </li>
+      </ul>
+    </div>
+    <div class="modal-img" style="background-image:url(${modalProject[index].img})">
+    
+    </div>
+
+    <div class="container">
+    <div class="modal-description-container">
+      <p class="modal-description">${modalProject[index].description}</p>
+    </div>
+    
+    <div class="modal-skills-container">
+    <ul class="modal-skills">
+    <li>
+      <p>html</p>
+    </li>
+    <li>
+      <p>css</p>
+    </li>
+    <li>
+      <p>javascript</p>
+    </li>
+  </ul>
+  <div class="modal-buttons-container">
+  <div class="modal-btn">
+    <a href="${modalProject[index].pageLink}">See live</a>
+    <i class="fas fa-external-link-alt"></i>
+  </div>
+  <div class="modal-btn">
+    <a href="${modalProject[index].sourcesLink}">See sources</a>
+    <i class="fab fa-github"></i>
+  </div>
+</div>
```

Comment:
- [ ] Please descrease the padding inside buttons
- [ ] Please add a devider line between the buttons and language tags

Your design  | Figma design
------------- | -------------
<img width="298" alt="Screenshot 2022-04-07 at 13 01 13" src="https://user-images.githubusercontent.com/77038610/162162471-b6cbeb1f-e6c9-4145-9ef7-540d46a64e55.png"> | <img width="269" alt="Screenshot 2022-04-07 at 13 00 54" src="https://user-images.githubusercontent.com/77038610/162162420-2d4dea18-f70a-4281-bcff-cabd6cc2dd12.png">